packages/sanity/src/structure/comments/src/hooks/index.ts

```diff
@@ -1,10 +1,10 @@
+export * from '../../../../core/studio/commentsSetup/useCommentsSetup'
```

Comment:
Do we want to re-export this hook from here?

Reply:
Nope, thanks for catching!